cmd/telemetrygen/config.go

```diff
@@ -35,8 +39,7 @@ var tracesCmd = &cobra.Command{
 	Short:   "Simulates a client generating traces",
 	Example: "telemetrygen traces",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		err := errors.New("[WIP] The 'traces' command is a work in progress, come back later")
-		return err
+		return traces.Start(tracesCfg)
```

Comment:
Could you document on `cmd/telemetrygen/README.md` how the traces command works now?